src/collectors/windows.plugin/metadata.yaml

```diff
@@ -1752,6 +1752,32 @@ modules:
                  DEALLOCATE nd_user_cursor
                  GO
                  ```
+
+              3.  **Configure SQL Server Network Settings**
+
+                 Enable SQL Server to accept TCP connections:
+
+                - Open `SQL Server Configuration Manager`
+                - Expand `SQL Server Network Configuration`
+                - Select `Protocols for <instance name>` in the console panel
+                - Double-click the `TCP` protocol in the details panel and set `Enabled` to `Yes`
+                - Go to the `IP Address` tab and locate the `IPAII` section:
+                  - Clear any value from the `TCP Dynamic Ports` field
+                  - Enter a port number in the `TCP Port` field (default is `1433`)
+                - Select `SQL Server Services` and restart your SQL Server instance
+
+                These steps must be performed for each SQL Server instance you want to monitor.
```

Comment:
Which steps? There are 4 steps (1, 2, 3 and 4).

Reply:
All the steps in section 3.

Reply:
What about 4 (Configure SQL Server Authentication)?

Reply:
This is optional.  Users only should configure it when they allow authentication with MSSQL users.

Reply:
But if they do - do they need to do it for each instance they want to monitor? I am trying to understand if "these steps must be performed for each SQL Server instance you want to monitor" is applicable for step 3 only or for 3 and 4.